straxen/plugins/__init__.py

```diff
@@ -38,3 +38,6 @@
 
 from . import online_monitor
 from .online_monitor import *
+
+from . import s2cnn_processing
```

Comment:
Hi Andrii, can you please put this functionality in Peak processing? I think that is the appropriate place.

Reply:
I am a bit afraid that peak processing file grow quiet large, especially when we will have multiple position reconstruction options. It is easy to copy-paste it there, but I think it is better to keep it separate for maintenance and further  developments.